reducer-todo/src/components/TodoComponents/TodoList.js

```diff
@@ -0,0 +1,23 @@
+import React, { useReducer } from 'react';
+import Todo from './Todo';
+import { todoReducer, initialState} from '../../reducers/Reducer'
+
+
+
+const TodoList = () => {
+
+    const [state, dispatch] = useReducer(todoReducer, initialState);
```

Comment:
I know I told you that you can put the reducer here, but it might actually be easier to have it at the App level (it probably needs to exist there for the form functionality anyway).